mne/io/eeglab/eeglab.py

```diff
@@ -204,8 +206,10 @@ def read_epochs_eeglab(input_fname, events=None, event_id=None, montage=None,
         Names or indices of channels that should be designated EOG channels.
         If 'auto', the channel names containing ``EOG`` or ``EYE`` are used.
         Defaults to empty tuple.
-    verbose : bool, str, int, or None
+    verbose : bool | str | int | None
         If not None, override default verbose level (see mne.verbose).
+    uint16_codec : str | None
+        Codec to use for uint16 char arrays (defaults to system default codec).
```

Comment:
I think this docstring should change actually - I just copied it from scipy. `uint16_codec` is only useful when the set file you are trying to load has some non-ascii characters and default scipy.io.loadmat fails with buffer error. see https://github.com/scipy/scipy/issues/5516